<launch>
  <include file="$(find neuro_model)/launch/neuro_model.launch"/>

  <include file="$(find neuro_arm)/launch/move_group.launch"/>
  
  <node name="left_arm_trajectory_pub" pkg="neuro_bridge" type="left_arm_trajectory_pub" output="screen">
    <!-- Reduce planed speed by this factor -->
  	<param name="deceleration" value="1.0"/>
  </node>

  <node name="left_gripper_trajectory_pub" pkg="neuro_bridge" type="left_gripper_trajectory_pub" output="screen"/>

  <node name="right_arm_trajectory_pub" pkg="neuro_bridge" type="right_arm_trajectory_pub" output="screen"/>

  <!-- Performing forward kinetic grasp and moving arm and gripper by voice control -->
  <node name="neurobot_arm_left" pkg="neuro_arm" type="neurobot_arm_left.py" respawn="true" output="screen">
    <!-- Topics for communication -->
    <param name="ctrl_vision_grasp_pose" value="/ctrl/vision/grasp/pose" />
    <param name="ctrl_vision_detect_table" value="/ctrl/vision/detect/table" />
    <param name="ctrl_voice_arm_left" value="/ctrl/voice/arm/left" />
    <param name="feed_arm_grasp_result" value="/feed/arm/left/move/result" />
    <!-- If using forward kinetic, notice that when using the param with rospy.getparam: -->
    <!-- add `~` before the param name like this: rospy.get_param('~use_fk') -->
    <param name="use_fk" value="true" />
    <param name="link_to_foot" value="0.44" />
  </node>

</launch>
